scripts/dts/edtlib.py

```diff
@@ -823,8 +824,7 @@ def depends_on(self):
     @property
     def enabled(self):
         "See the class docstring"
-        return "status" not in self._node.props or \
-            self._node.props["status"].to_string() != "disabled"
+        return self._node_status
```

Comment:
Could remove this function and rename `Node._node_status` to `Node.enabled`.

Reply:
Drop property and add public instance variable Node.enabled